### PR TITLE
Add next steps to registration guide

### LINKED ISSUE #1806: Add step to move LSK to sidechain
### Description

- Add additional step: transfer LSK token from MC to SC to pay for CCM transaction fees.
- Add cross-chain token transfer command's usage as well.

### Affected branch / URL
`development`

### Page location
`build-blockchain/register-sidechain.adoc`

### Additional resources
https://lisk.com/documentation/beta/understand-blockchain/interoperability/index.html

### LINKED ISSUE #1815: Add offchain-registration steps to guide
### Description
Describe how to provide offchain app metadata for UIs

#### Affected branch / URL
`development`

#### Page location
`build-blockchain/register-sidechain.adoc`

### Additional resources
https://github.com/LiskHQ/app-registry#lisk-application-registry

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1815 
Closes #1806

### CODE REVIEW

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc [hunk 1]
```diff
@@ -2181,3 +2189,183 @@ This should return the mainchain validators data stored in the xref:{url_sidecha
...
+│   │   └── tokens
+│   │   │   ├── token.png
+│   │   │   └── token.svg
+tsconfig.json
+----
+
+where `<APP>` is the name of the sidechain app.
+
+`app.json`::
+Off-chain metadata of a sidechain, to be used by user frontends like Lisk Wallet.
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]:
```suggestion
Off-chain metadata of a sidechain, to be used by user frontends such as the Lisk Wallet.
```

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc [hunk 2]
```diff
@@ -2181,3 +2189,183 @@ This should return the mainchain validators data stored in the xref:{url_sidecha
...
+
+`images/application/`::
+Icons to be used for the sidechain application.
+
+`images/tokens/`::
+Icons to be used for the native tokens of the sidechain.
+
+=== Move LSK to sidechain
+
+NOTE: This step is not necessary, if the default token for CCM fees (LSK) is changed to a native sidechain token.
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]:
```suggestion
NOTE: This step is not necessary, if the default token for the CCM fees (LSK) is changed to a native sidechain token.
```

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc [hunk 3]
```diff
@@ -2181,3 +2189,183 @@ This should return the mainchain validators data stored in the xref:{url_sidecha
...
+
+To be able to pay the CCM transaction fees, it is necessary to move a sufficient amount of LSK tokens from mainchain to sidechain via a cross-chain transfer transaction.
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc:2331]:
```suggestion
To be able to pay the CCM transaction fees, it is necessary to move a sufficient amount of LSK tokens from the mainchain to the sidechain via a cross-chain transfer transaction.
```

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc [hunk 4]
```diff
@@ -2181,3 +2189,183 @@ This should return the mainchain validators data stored in the xref:{url_sidecha
...
+* `messageFee`: Enter the fee for the CCM.
+* `messageFeeTokenID`: Enter the token ID of the token that should be used to pay the fee.
+Enter here the tokenID of the LSK token.
+
+NOTE: Sending CCMs from mainchain to sidechain does not <<activate-the-sidechain>>, yet.
+Only a CCM from sidechain to mainchain will enable the xref:{url_sidechain_reg_recovery_liveness}[liveness condition].
+
+== Activate the sidechain
+
+After registering the sidechain and setting up the required relayer nodes, the sidechain is ready to be xref:{url_sidechain_reg_recovery_active}[activated] .
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]:
```suggestion
After registering the sidechain and setting up the required relayer nodes, the sidechain is ready to be xref:{url_sidechain_reg_recovery_active}[activated].
```

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc [hunk 5]
```diff
@@ -2181,3 +2189,183 @@ This should return the mainchain validators data stored in the xref:{url_sidecha
...
+
+To activate the sidechain, send a xref:{url_sidechain_reg_recovery_1stccu}[first CCU] from sidechain to mainchain.
+
+[CAUTION]
+====
+* The first cross-chain update (CCU) to be submitted on the mainchain must contain a certificate *less than 15 days old*.
+* Once the sidechain is *active*, it needs to fulfil the xref:{url_sidechain_reg_recovery_liveness}[liveness condition], otherwise it will be *terminated*.
+====
+
+Once the first CCU was sent successfully from sidechain to mainchain, the sidechain is now activated and interoperable with the mainchain and other sidechains the Lisk ecosystem.
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]:
```suggestion
Once the first CCU was sent successfully from sidechain to mainchain, the sidechain is now activated and interoperable with the mainchain and other sidechains within the Lisk ecosystem.
```

**Review by chris529 — COMMENTED** [review]
Looks good.. Just a few minor points.. 👍

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc [hunk 6]
```diff
@@ -2181,3 +2189,183 @@ This should return the mainchain validators data stored in the xref:{url_sidecha
...
+? Please enter: amount:  1000000000
+? Please enter: receivingChainID:  01000001
+? Please enter: recipientAddress:  lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu
+? Please enter: data:  cross-chain transfer
+? Please enter: messageFee:  1
+? Please enter: messageFeeTokenID:  0100000000000000
+----
+
+* `tokenID`: The token ID of the LSK token.
+* `receivingChainID`: Enter the chain ID of the respective sidechain you want to send LSK to.
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]:
```suggestion
* `receivingChainID`: Enter the chain ID of the respective sidechain where you intend to transfer your LSK tokens.
```

**Review by chris529 — APPROVED** [review]
LGTM! 👍 I just left one minor suggestion..

**Review by sitetester — APPROVED** [review]

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc [hunk 7]
```diff
@@ -2181,3 +2189,183 @@ This should return the mainchain validators data stored in the xref:{url_sidecha
...
+
+To facilitate cross-chain communication, at least one relayer node is required on the mainchain and the sidechain.
+
+=== Setup relayer nodes on mainchain and sidechain
+
+Please check the guide xref:{url_run_relayer}[] for a step-by-step explanation how to turn a node into a relayer.
+
+=== Register off-chain data in the app-registry
+
+As a last step, it is recommended to register the sidechain related off-chain metadata to the {url_github_appregistry}[app registry^].
```
Comment (ManuGowda) [inline docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]:
```suggestion
As a last step, it is necessary to register the sidechain related off-chain metadata to the {url_github_appregistry}[app registry^].
```

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc [hunk 8]
```diff
@@ -2181,3 +2189,183 @@ This should return the mainchain validators data stored in the xref:{url_sidecha
...
+│   │   └── tokens
+│   │   │   ├── token.png
+│   │   │   └── token.svg
+tsconfig.json
+----
+
+where `<APP>` is the name of the sidechain app.
+
+`app.json`::
+Off-chain metadata of a sidechain, to be used by user frontends such as the Lisk Wallet.
```
Comment (ManuGowda) [inline docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]: user frontends? maybe `to be used by client applications such as Lisk Desktop, Mobile and other explorers`?

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc [hunk 9]
```diff
@@ -2181,3 +2189,183 @@ This should return the mainchain validators data stored in the xref:{url_sidecha
...
+
+.Example: app.json
```
Comment (ManuGowda) [inline docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc:2232]: maybe mention the README of app-registry? which already explains the end to end step in detail?

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**